libevmasm/Assembly.h

```diff
@@ -158,6 +160,16 @@ class Assembly
 
 	bool isCreation() const { return m_creation; }
 
+	void setSources(std::vector<std::shared_ptr<std::string const>> _sources) {
```

Comment:
Coding style.

Could you split this up further and only include the change to `Assembly::assemblyJSON` - maybe not even the `_includeSourceList` parameter?

Reply:
good idea!